src/sickle.ts

```diff
@@ -302,13 +307,13 @@ class Annotator {
       }
       return;
     }
-    if (skipComments) {
-      this.fail('skipComments unimplemented for complex Nodes');
-    }
     let lastEnd = node.getFullStart();
+    if (skipComments) lastEnd = node.getStart();
```

Comment:
I have a preference for single assignments (you can use const then too):
lastEnd = skipComments ? node.getStart() : node.getFullStart()

but really up to you.